notifications/2024-11-24_CHG0064530.md

```diff
@@ -0,0 +1,29 @@
+Hi @all, your attention is needed! 
+
+**What is happening?**
+
+CHG0064530 STMS NET - CAL - Arista EOS Upgrade - Calgary Network Switches - MSP01/MSP02/MEL01/MEL02
+
+Network switches in the core of the Calgary datacenter will be upgraded.
+
+**When?**
+
+Starting at Sunday Nov 24th at 6:00am completing before 9:00am
+
+**Will there be an impact on the Platform apps?**
+
+No, no downtime is expected at the time, the switches are redundant.
```

Comment:
```suggestion
No downtime is expected at the time, the switches are redundant.
```
Seems cleaner this way. If there's a reason for the original formatting, then don't change. Will approve regardless.

Reply:
Copying from the example in the template https://github.com/BCDevOps/platform-services-status-page-notifications/blob/a505e15cb7334dad496d5cb48037b5b5414ddca3/maintenance_template.md?plain=1#L19

Trusting that the recent review of the template from Pilar has the most correct grammar.